tests/main/snap-seccomp-blocks-tty-injection/task.yaml

```diff
@@ -24,6 +24,6 @@ execute: |
         apparmor_parser -r /var/lib/snapd/apparmor/profiles/snap.test-snapd-sh.sh
     fi
 
-    snap run test-snapd-sh.sh -c "\$SNAP_COMMON/test-tiocsti"   < /dev/tty1 2>&1 | MATCH 'normal TIOCSTI: -1 \(Operation not permitted\)'
-    snap run test-snapd-sh.sh -c "\$SNAP_COMMON/test-tiocsti"   < /dev/tty1 2>&1 | MATCH 'high-bit-set TIOCSTI: -1 \(Operation not permitted\)'
+    snap run test-snapd-sh.sh -c "\$SNAP_COMMON/test-tiocsti"   < /dev/tty1 2>&1 | MATCH 'normal TIOCSTI: -1 \((Operation not permitted|Permission denied)\)'
```

Comment:
`TIOCSTI` and `TIOCLINUX` get `EACCESS. `(1UL<<32)|TIOCSTI` and `(1UL<<32)|TIOCLINUX` get `EPERM`. Should we just change `snap-seccomp-blacklist` to use `EACCESS`  for those cases like `snap-seccomp`?

Otherwise we should explain here why we get either error depending on the architecture.

Reply:
That is a good point. Not sure if @mvo5 thinks it is too late to change this now but at least adding a comment here to explain the 32-vs-64 bit difference would be useful for the future.

Reply:
Adding a comment would be good, I have no strong opinion about changing it to be consistent but AIUI we are already inconsistent(?) in which case I think we should not change it here but maybe later.